design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,658 @@
+as `forProvider`. The `initProvider` fields would be merged with the
+`forProvider` fields during the `Create` step of the managed resource
+reconciliation loop. Fields that are specified in both `initProvider` and
+`forProvider` give precedence to the `forProvider` fields.
+
+```yaml
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: NodeGroup
+metadata:
+  name: my-group
+spec:
+  initProvider:
+    scalingConfig:
+      desiredSize: 1
+  forProvider:
+    region: us-east-1
+    scalingConfig:
+      maxSize: 5
+      minSize: 1
+```
+
+This would allow users to specify fields that would only be used on creation,
+but not on updates.
+
+When it comes to the implementation, the handling would all need to be done
+on the provider level. The `initProvider` fields should only be used in the
+`Create` step of the managed resource reconciliation loop.
+
+Support for `initProvider` would need to be a manual change in each provider.
+From our side we could provide a helper function to merge the `initProvider`
+and `forProvider` fields which would be used in the `Create` step.
+
+### Feature Gating
+
+The new management policy would use the same feature gate already introduced
+by `ObserveOnly`. So it would be an alpha feature that will be off by default
+and will be controlled by `--enable-alpha-management-policies` flag in
+Providers. It will also enable the `initProvider` feature.
+
+### Examples how the new functionality would affect the use cases
+
+Some use cases mentioned in the beginning of the document would be
+solved just by omitting the management policy `LateInitialize`. For
+others, the `initProvider` field would need to be used, mostly because
+the fields in question are required on creation.
+
+#### AutoScalingGroup
+
+The `desiredCapacity` is not required on creation, but populated by late
+initialization. It can be solved just by omitting `Late Initialize` management
+policy.
+
+```yaml
+spec:
+  managementPolicy: ["Create", "Update", "Delete", "Observe"]
+  forProvider:
+    maxSize: 5
+    minSize: 1
+    launchConfigurationNameRef:
+      name: sample-launch-config
+    ...
+```
+
+#### DynamoDB Table
+
+The `readCapacity` and `writeCapacity` are not required, but they are required
+if the `billingMode` is set to `PROVISIONED`. So the `initProvider` field
+would need to be used alongside omitting `LateInitialize` management policy.
+
+```yaml
+spec:
+  managementPolicy: ["Create", "Update", "Delete", "Observe"]
+  initProvider:
+    readCapacity: 1
+    writeCapacity: 1
+  forProvider:
+    billingMode: PROVISIONED
+...
+```
+
+#### EKS NodeGroup
+
+The `scalingConfig.desiredSize` is required so `initProvider` would need
+to be used alongside omitting `LateInitialize` management policy. This way
+the autoscaler would be able to control the `desiredSize` after creation.
+
+```yaml
+spec:
+  managementPolicy: ["Create", "Update", "Delete", "Observe"]
+  initProvider:
+    scalingConfig:
+      desiredSize: 1
```

Comment:
It looks like we are disabling lateInit whenever we set `initProvider`. So, I am wondering if this is something that could be generalized as, if a field is provided under initProvider, never late initialize that specific parameter even managementPolicy contains lateInit. I think the underlying idea is providing a field under `initProvider` implies that it should not appear under `forProvider`.

Reply:
We could add such a functionality that the late initialization code checks for `initProvider` fields when updating the `forProvider` fields. It already does it for `forProvider` fields, so it would not be a stretch.